README.md

```diff
@@ -60,9 +60,17 @@ pgai.install(DB_URL)
 
 # Quick Start
 
-This quickstart demonstrates how pgai Vectorizer enables semantic search and RAG over PostgreSQL data by automatically creating and synchronizing embeddings as data changes.
+## Provider-Specific Quick Start Guides
+For detailed guides tailored to specific embedding providers:
+- [Complete Documentation](/docs/README.md)
+- [Quick Start with Ollama](/docs/vectorizer/quick-start.md)
+- [Quick Start with OpenAI](/docs/vectorizer/quick-start-openai.md) 
+- [Quick Start with VoyageAI](/docs/vectorizer/quick-start-voyage.md)
```

Comment:
are these quickstarts up-to-date? (I doubt it). Also we should link to the fastapi example here too?